include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -736,12 +736,22 @@ __parallel_find_or(_ExecutionPolicy&& __exec, _Brick __f, _BrickTag __brick_tag,
                                           sycl::range</*dim=*/1>(__wgroup_size)),
                 [=](sycl::nd_item</*dim=*/1> __item_id) {
                     auto __local_idx = __item_id.get_local_id(0);
-
+#if !_ONEDPL_SYCL2023_ATOMIC_REF_PRESENT
                     // connect global atomic with global memory
                     sycl::atomic<_AtomicType> __found(__temp_acc.get_pointer());
                     // connect local atomic with local memory
                     sycl::atomic<_AtomicType, sycl::access::address_space::local_space> __found_local(
                         __temp_local.get_pointer());
+#else
+                    // connect global atomic with global memory
+                    sycl::atomic_ref<_AtomicType, sycl::memory_order::relaxed, sycl::memory_scope::work_group,
+                                     sycl::access::address_space::global_space>
+                        __found(__temp_acc[0]);
```

Comment:
Does it make sense to define an alias in `__dpl_sycl` namespace instead?

Reply:
fixed

Reply:
I think no, because we have only one place with sycl::atomic or sycl::atomic_ref and different parameters in their constructors...

Reply:
I will think about this, thanks

Reply:
Fixed